tests/unit/test_pynblint.py

```diff
@@ -1,6 +1,154 @@
 from pynblint.notebook import Notebook
 from pathlib import Path
+import pytest
 
 if __name__ == '__main__':
-    nb = Notebook(Path('../fixtures/my-attempt-at-analytics-vidhya-job-a-thon.ipynb'))
-    print(nb.get_pynblint_results())
+    pytest.main()
```

Comment:
If I'm not mistaken, we don't need this statement (and - more in general - there's no need to have a "main" in a test module). By directly executing `pytest`, the tests contained in this module should be discovered and run (as the names of the module itself and of the functions contained therein are prefixed with the word `test_`). 

In our case, from the root folder of the project, we should be able to invoke pytest this way:

```bash
pytest tests/
```

Reply:
I am aware of that. But since you wrote the main I thought you wanted to use it. For example with ```python test_pynblint.py```. If you don't, I can erase it.